src/test/java/HomeWork16.java

```diff
@@ -0,0 +1,34 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class HomeWork16 extends BaseTest {
+
+    @Test
+    public void registrationNavigation() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+        WebElement registrationButton = driver.findElement(By.cssSelector("a[href='registration']"));
+        registrationButton.click();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        WebElement registrationLink = driver.findElement(By.cssSelector("input[type=email]"));
```

Comment:
//Comment from HW team
Hello,
I have reviewed your homework.
Great job on creating your Homework 16 and implementing Selenium WebDriver

But in homework 16, in step 6 required to ```6. Verify that you are redirected to Registration page using Assert method.```
I am currently seeing you verify in this step is check email field isDisplayed(). It is not correct. Because both Login page and Registration page have email field, so if you verify it will not be possible.

Key points:
check url is https://qa.koel.app/registration or Register new account or Reset your password text is displayed.
//Example:
```
String regURL = "https://qa.koel.app/registration";
        Assert.assertEquals(driver.getCurrentUrl(), regURL);
```